products/urls.py

```diff
@@ -1,9 +1,10 @@
 from django.urls import path
 
-from products.views import CategoryView,ProductView
+from products.views import CategoryView,ProductView,ProductFilterView
 
 urlpatterns = [
     path('/category', CategoryView.as_view()),
     path('/product', ProductView.as_view()),
+    path('/productfilter', ProductFilterView.as_view())
```

Comment:
```suggestion
    path('/filter', ProductFilterView.as_view())
```
여기도 중복 문자열은 줄이는게 좋겠죠

Reply:
productfilter 삭제 했습니다~